src/main/java/lotto/domain/model/Lotto.java

```diff
@@ -0,0 +1,74 @@
+package lotto.domain.model;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import lotto.domain.Match;
+
+public record Lotto(List<Integer> numbers) {
+    
+    public static final String SPACE = " ";
+    public static final String EMPTY = "";
+    public static final String COMMA = ",";
+    public static final int MIN_LOTTO_NUMBER = 1;
+    public static final int MAX_LOTTO_NUMBER = 46;
+    public static final int INITIAL_MATCH_COUNT = 0;
+    
+    public Lotto() {
+        this(generateLottoNumberByPay());
+    }
+    
+    public Lotto(String lottoNumbers) {
+        this(generateLottoNumberByInput(lottoNumbers));
+    }
+    
+    public Lotto(int numA, int numB, int numC, int numD, int numE, int numF) {
```

Comment:
```suggestion
    public Lotto(int... nums) {
```
가변인자로 구현 가능함
이번 기회에 가변인자 추가학습할 것을 추천